esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -1603,34 +1603,55 @@ static int bindingCompareFunc(IInterface * const *_itm1, IInterface * const* _it
         return 0;
 }
 
+
+void getAllEspProcesses(StringArray& processes)
```

Comment:
just noticed the logic calling this method depends on the result being sorted, I strongly suggest we decouple that dependency, or at least make it obvious that this method returns sorted alphabetically ( I can see someone many years from now, removing the sort statement otherwise)